workshop/SCHEDULE.md

```diff
@@ -56,32 +56,34 @@ Minneapolis, MN 55414 in Room 105.
 |-------------|------------------------------------------------|
 | **Day 3**   | **2024-08-21 <br> Bulk RNA-Seq**  |
 | 8:30 AM     | Breakfast and coffee available |
-| 9:00 AM     | RNA-Seq quantification (trim, filter, quant) |
-| 10:00 AM    | RNA-Seq import with tximeta |
+| 9:00 AM     | [RNA-Seq quantification (trim, filter, quant)](../completed-notebooks/RNA-seq/01-qc_trim_quant.nb.html) |
+| 10:00 AM    | [RNA-Seq import with tximeta](../completed-notebooks/RNA-seq/02-gastric_cancer_tximeta.nb.html) |
 | 10:45 AM    | Coffee break |
-| 11:00 AM    | RNA-Seq exploratory analysis |
-|             | Exercise: Neuroblastoma cell line tximeta |
-|             | Exercise: Exploratory data analysis |
+| 11:00 AM    | [RNA-Seq exploratory analysis](../completed-notebooks/RNA-seq/03-gastric_cancer_exploratory.nb.html) |
+|             | **TODO this isn't an exercise anymore?** [Exercise: Neuroblastoma cell line tximeta](../completed-notebooks/RNA-seq/04-nb_cell_line_tximeta.nb.html) |
```

Comment:
This is an "in class exercise." The link you have here is to the instructions which is fine, but the "completed" notebook is in the `setup` directory and should probably go into magic files.

Reply:
It should already be in magic files, but let me know if that's not true (https://github.com/AlexsLemonade/training-admin/issues/1132#issuecomment-2274039079).

Reply:
Ah, you don't mean the output necessarily - I see.

Reply:
> the "completed" notebook is in the setup directory and should probably go into magic files.

I've copied our `RNA-seq/setup/ref_notebooks/04*` notebooks over, (for now?) living in `/shared/data/training-data/2024-august/RNA-seq/ref-notebooks_nb_cell_line`. Eh?

Reply:
I guess we probably only need the `Rmd` copied over but wasn't 100% sure, so did both for now.